.pylintrc

```diff
@@ -135,7 +135,6 @@ disable=print-statement,
         exception-escape,
         comprehension-escape,
         line-too-long,               # we don't enforce some fixed line length but rather complain about long lines in reviews.
-        cyclic-import,               # landscape re-reports them every time, https://github.com/PyCQA/pylint/issues/59
```

Comment:
this is another thing i don't really understand. we're not using landscape anymore, you introduced this a few commits before and remove it here.
so do we want to check for cyclic imports? could you then not add it in the other commit or did i miss something?